Feature/#193 matsuriba max

dassai0079 opened the pull request:
close #193

dassai0079 committed:
matsuriba-max-2025/index.md の作成

dassai0079 committed:
まえがき

dassai0079 committed:
参加経緯

dassai0079 committed:
finish?

dassai0079 committed:
Update index.md

dassai0079 committed:
add 20250301_134208.jpg

dassai0079 committed:
改行を入れて見やすく

dassai0079 committed:
index.md の更新

dassai0079 committed:
表現修正

dassai0079 committed:
日付変更

fjktkm commented on content/post/matsuriba-max-2025/index.md:
```diff
@@ -0,0 +1,153 @@
+---
+title: "NITMic MatsuribaMax参加録"
```
NITMic の記事なので NITMic とわざわざ書く必要はない
今年度も行われる可能性があるので年号は入れたほうがいい
正式名称準拠で Matsuriba と MAX の間にはスペースを入れたほうがいい
「Matsuriba MAX 2025 に出展しました」とかが無難そう

dassai0079 replied:
Commit 17eeafe にて修正しました

fjktkm commented on content/post/matsuriba-max-2025/index.md:
```diff
@@ -0,0 +1,153 @@
+thumbnail: "20250301_134208.jpg"
+categories: ["記事"]
+description: "2025年3月1日開催のMatsuribaMaxにNITMicがブース出展を行いました"
+tags: ["NITMic", "2025", "外部イベント", "日記"]
```
日記なの？かな？
「Matsuriba Tech」は入れておいていいかも

dassai0079 replied:
Commit 17eeafe にて修正しました

fjktkm commented on content/post/matsuriba-max-2025/index.md:
```diff
@@ -0,0 +1,153 @@
+
+はじめまして、2025 年度部長の 獺祭 です。
+
+さる2025年3月、私たちはStationAiにて開催されたMatsuribaMAX2025にブースを出展させていただきました。
```
表記は「STATION Ai」「Matsuriba MAX 2025」のように正式名称に準拠してください

dassai0079 replied:
Commit 65a9dfeb にて修正しました

fjktkm commented on content/post/matsuriba-max-2025/index.md:
```diff
@@ -0,0 +1,153 @@
+
+はじめまして、2025 年度部長の 獺祭 です。
+
+さる2025年3月、私たちはStationAiにて開催されたMatsuribaMAX2025にブースを出展させていただきました。
```
させていただく構文は公式な文書なので避けたいところです
「出展いたしました」とかで十分丁寧さは伝わると思います

dassai0079 replied:
Commit 2d0a9c7 にて修正しました

fjktkm commented on content/post/matsuriba-max-2025/index.md:
```diff
@@ -0,0 +1,153 @@
+
+### NITMicとMatsuriba
+そもそもMatsuribaというのは、NxTEND様が主催している東海地方の技術系サークル・団体向け交流イベントです。
+MatsuribaMAXはその拡大版で、外部企業の方をお呼びしたり、制作物を展示するブースを設置したりして盛大にやろうぜ！！というイベントです。
```
「Matsuriba MAX」に修正してください

dassai0079 replied:
Commit 65a9dfe にて修正しました

fjktkm commented on content/post/matsuriba-max-2025/index.md:
```diff
@@ -0,0 +1,153 @@
+### 都合のいい条件その1:開催場所がめちゃくちゃ近い
+
+これ、かなりデカかったです。
+2024は名古屋国際会議場で開催されましたが、2025は完成間もないStationAiで開催されました。
```
「STATION Ai」に修正してください

dassai0079 replied:
Commit 65a9dfe にて修正しました

fjktkm commented on content/post/matsuriba-max-2025/index.md:
```diff
@@ -0,0 +1,153 @@
+2024は名古屋国際会議場で開催されましたが、2025は完成間もないStationAiで開催されました。
+これがまた鶴舞公園を挟んですぐそばと、名工大にめちゃくちゃ近いんです。
+ということは、部員に交通費を負担してもらう必要が無くなるわけです。
+しかも、物品の搬入だって徒歩でできる。(しかし、これは後で失敗だったと分かります)
```
括弧書きは「。」の前に書くのが一般的です
つまり，「しかも、物品の搬入だって徒歩でできる（しかし、これは後で失敗だったと分かります）。」という感じです

dassai0079 replied:
Commit 2d0a9c7 にて修正しました

fjktkm commented on content/post/matsuriba-max-2025/index.md:
```diff
@@ -0,0 +1,153 @@
+
+## 開催当日
+
+当日、私は同じくブース参加するC0de(名工大プログラミング部)が車で機材の運び出しするというので、機材ともども便乗させていただくことにしました。
```
地の文が日本語のときは全角括弧のほうが好ましいと思います（強制はしません）

dassai0079 replied:
Commit 2d0a9c7 にて修正しました

fjktkm commented on content/post/matsuriba-max-2025/index.md:
```diff
@@ -0,0 +1,153 @@
+{{< figure link="20250301_134208.jpg" caption="ブースの画像" >}}
+
+ロールアップバナーは部屋の中で結構な存在感を放っていたと思います。なんせでかいからね！
+その他にも賑やかしグッズを大量に持って行ったので、他団体よりも良い意味で圧倒的にうるさいブースだったと思います。(ゲームの音声もあったから物理的にもうるさかったかも)
```
括弧書きは「。」の前に移動してください

dassai0079 replied:
Commit 2d0a9c7 にて修正しました

fjktkm commented on content/post/matsuriba-max-2025/index.md:
```diff
@@ -0,0 +1,153 @@
+こんなこともありましたが、ブースはありがたいことに盛況でNITMicのアピールには成功したと思います。
+時間はあっという間に過ぎてゆき、ブース撤収を命じられますが、スポンサー企業紹介の直前でした。
+このまま名工大に運び出しを行っていたのでは間違いなく聞けません。
+仕方ないので、NITMicブースの物品を部室に運ぶ段取りを進めますが、他団体はStationAi内の倉庫に運び出してさっさと発表会場に向かっていきます。
```
「STATION Ai」に修正してください

dassai0079 replied:
Commit 65a9dfe にて修正しました

fjktkm commented on content/post/matsuriba-max-2025/index.md:
```diff
@@ -0,0 +1,153 @@
+当初の目的は無事に果たされたようで、私は満足しています。
+
+## 後日
+実は、交流会でMatsuribaのえらい人からNITMicのMatsuribaへの加盟を打診されていました。
```
正式には「Matsuriba Tech」だと思うので略さず正式名称を使ったほうがいいと思います

dassai0079 replied:
Commit 65a9dfe, Commit 498f139 にて修正しました

fjktkm commented on content/post/matsuriba-max-2025/index.md:
```diff
@@ -0,0 +1,153 @@
+また、これがきっかけか分かりませんが、外部イベントに参加する部員も増えたようです。
+当初の目的は無事に果たされたようで、私は満足しています。
+
+## 後日
```
後日談？

dassai0079 replied:
Commit 2d0a9c7 にて修正しました

fjktkm commented on content/post/matsuriba-max-2025/index.md:
```diff
@@ -0,0 +1,153 @@
+このせいか、寝不足のせいかは分かりませんが、交流会ではぐでんぐでんに酔いつぶれてしまいました。
+全く見苦しい所を見せてしまった...
+
+## 総括
```
「はじめに」で始まっているので「おわりに」とかが無難かな
「総括」って公の文書で使っているのはあまり見かけないと思う

dassai0079 replied:
Commit 2d0a9c7 にて修正しました

fjktkm commented on content/post/matsuriba-max-2025/index.md:
```diff
@@ -0,0 +1,153 @@
+
+## 後日
+実は、交流会でMatsuribaのえらい人からNITMicのMatsuribaへの加盟を打診されていました。
+そして、部員からの賛同の上でこの打診を受諾いたしました。
```
打診・受諾という表現はやや引っ掛かります
向こうの組織の面目もあるので，もう少しオブラートに包んでもいいかも？

dassai0079 replied:
Commit 2d0a9c7 にて修正しました

fjktkm reviewed (changes requested):
コメント入れたので確認お願いします

fjktkm commented on content/post/matsuriba-max-2025/index.md:
```diff
@@ -0,0 +1,153 @@
+---
+title: "NITMic MatsuribaMax参加録"
+date: "2025-8-30"
```
"2025-08-30" にしないとうまく認識されないみたいです

dassai0079 replied:
Commit 17eeafe にて修正しました

fjktkm commented on content/post/matsuriba-max-2025/index.md:
```diff
@@ -0,0 +1,153 @@
+適当にうろうろしていると開場時間となりました。
+
+せっせとブース設営を行います。出来上がったブースはこのような感じ。
+{{< figure link="20250301_134208.jpg" caption="ブースの画像" >}}
```
```
{{< load-photoswipe>}}
{{< gallery >}}
{{< figure link="20250301_134208.jpg" caption="ブースの画像" >}}
{{< /gallery >}}
```
のようにしてください

dassai0079 replied:
Commit 1e86ed7 にて修正しました

dassai0079 committed:
fix:正式名称との表記ゆれ

- MatsuribaMAX2025 → Matsuriba MAX 2025
- StationAi → STATION Ai

dassai0079 committed:
fix:meta data

- date修正
- タグ修正
- タイトル修正
- description修正

dassai0079 committed:
fix:日本語表現

- カッコを全角に
- カッコと句点(。)の順序を修正
- "させていただく"表記
- 後日談でのやり取り
- 見出しのタイトル
- 年(2025,2024)の表記

dassai0079 committed:
fix: Matsuriba Tech表記への修正漏れ

dassai0079 committed:
fix:英字と日本語の間に半角スペース

prettierかけて数字の前のスペースとっただけ

dassai0079 committed:
画像貼り付けコードを修正

#261 のレビューに基づいて
 https://github.com/nitmic/nitmic.club.nitech.ac.jp/pull/261#discussion_r2313294613

fjktkm committed:
Fix minor style issues

fjktkm reviewed (approved):
LGTM
